manifests/config.pp

```diff
@@ -90,7 +90,7 @@
 
   if $ntp::disable_dhclient {
     augeas { 'disable ntp-servers in dhclient.conf':
-      context => '/files/etc/dhcp/dhclient.conf',
+      context => '/etc/dhcp/dhclient.conf',
```

Comment:
@eimlav Why was this changed? Maybe you meant to use `incl` instead of `context`? As it is, it seems to be currently broken.

Reply:
https://github.com/puppetlabs/puppetlabs-ntp/pull/521

Reply:
Hi @raphink. I'm not sure why I made this change... I'll put a fix up to revert it back!